pkg/subvolume/subvolume.go

```diff
@@ -0,0 +1,145 @@
+/*
+Copyright 2023 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package subvolume
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"os"
+	"strings"
+
+	exec "github.com/rook/kubectl-rook-ceph/pkg/exec"
+	"github.com/rook/kubectl-rook-ceph/pkg/k8sutil"
+	"github.com/rook/kubectl-rook-ceph/pkg/logging"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type fsStruct struct {
+	Name string
+	data []string
+}
+
+type subVolume struct {
+	fs     string
+	svg    string
+	subvol string
+}
+
+func List(ctx context.Context, clientsets *k8sutil.Clientsets, clusterNamespace, operatorNamespace, flag string) []subVolume {
```

Comment:
Name the params, otherwise it's difficult to know what flag it is. Also, a bool is preferred in an API like this.
```suggestion
func List(ctx context.Context, clientsets *k8sutil.Clientsets, clusterNamespace, operatorNamespace, includeStale bool) []subVolume {
```